src/scss/components/_card.scss

```diff
@@ -82,29 +67,18 @@
     }
   }
 
-  &__title {
-    color: get-color(blackNeutral);
-  }
-
-  &__header {
-    .card__title {
-      color: get-color(rastaGray);
-    }
-  }
-
   &__content {
     color: get-color((color: blackNeutral, opacity: .6));
     font-size: 14px;
     font-style: italic;
-
-    .card__header + & {
-      margin-top: 16px;
-    }
+    width: 100%;
```

Comment:
@felipeorlando @eduardojmatos Testing with diferent text sizes, required to explicit set the width